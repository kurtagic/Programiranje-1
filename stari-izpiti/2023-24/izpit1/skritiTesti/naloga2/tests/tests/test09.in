1 50 87 805
26 72 746
26 72 438
26 72 728
26 72 701
20 72 470
20 72 815
18 38 400
34 72 25
20 38 603
20 42 452
11 38 156
26 38 226
26 72 164
26 38 366
2 42 485
29 42 199
34 38 390
20 32 889
34 72 876
34 38 792
29 38 356
46 72 492
34 38 738
13 38 923
29 32 465
13 42 510
26 40 515
37 32 755
2 32 547
13 77 145
11 38 36
2 36 525
26 36 846
34 72 847
26 72 805
46 72 442
36 34 407
34 38 135
0 4 527
34 84 801
34 3 324
39 3 661
29 69 267
13 34 229
46 3 82
2 69 687
11 72 382
32 3 686
34 19 929
39 4 799
0 69 171
19 4 191
11 81 831
41 36 23
47 56 513
46 9 888
12 32 663
47 4 803
29 62 42
47 0 724
37 28 534
18 32 327
0 4 138
39 28 583
34 36 401
11 46 203
41 81 466
22 34 756
37 73 65
2 81 199
11 38 895
20 71 486
18 69 91
11 42 359
34 9 311
15 40 135
29 62 803
32 9 754
15 46 697
37 72 51
22 62 904
6 4 770
42 32 735
22 42 121
20 4 812
36 84 292
36 9 24
7 81 453
32 34 730
42 27 142
10 19 820
36 81 847
7 0 101
36 0 771
11 10 512
1 16 828
9 81 897
9 47 232
42 62 181
37 71 567
20 28 94
44 40 216
29 17 206
11 45 428
3 16 842
26 71 694
24 45 836
42 3 465
13 45 673
26 4 534
34 4 925
34 82 197
47 4 368
4 79 806
7 67 50
42 16 195
19 81 475
12 45 606
32 10 608
29 9 914
3 69 167
1 28 930
41 71 525
26 81 357
2 73 907
8 70 748
41 22 679
1 10 451
27 68 341
40 43 746
20 64 911
10 9 75
1 5 875
22 65 358
0 7 781
44 48 424
45 20 271
7 19 447
37 42 651
40 56 392
45 19 471
4 73 108
34 38 342
40 22 834
33 3 66
42 38 685
1 69 846
42 62 119
41 65 476
26 44 873
10 68 708
34 16 574
42 28 605
9 72 51
2 65 394
13 68 219
47 38 393
1 45 864
19 81 50
3 73 748
40 27 342
39 72 545
32 79 522
17 64 354
35 40 178
21 16 286
3 65 246
37 82 659
47 34 781
33 46 355
11 31 464
26 44 53
11 58 731
29 0 75
37 81 578
18 17 489
42 62 856
40 84 845
13 2 595
19 67 493
10 69 251
43 77 355
36 62 278
39 3 309
44 45 469
47 17 268
29 31 176
37 17 825
20 34 813
41 45 870
20 20 260
11 72 452
1 42 34
26 4 401
18 70 876
15 4 294
40 9 273
41 44 201
44 4 350
36 64 163
28 16 588
39 11 689
36 47 191
12 20 584
1 76 90
4 44 926
42 9 70
34 48 707
25 45 926
8 58 237
26 64 65
13 62 553
8 70 352
40 9 29
1 81 701
42 16 628
44 69 714
42 4 483
46 7 340
34 32 643
28 4 7
24 32 39
36 65 623
41 62 145
47 32 310
1 56 664
39 32 216
12 9 28
12 62 268
13 70 810
8 70 513
40 79 425
42 7 733
10 77 910
28 64 319
36 64 833
36 41 877
26 64 788
0 16 207
33 45 775
42 71 688
7 42 495
21 4 200
1 31 879
26 3 811
39 62 495
17 34 919
25 4 524
39 4 604
11 34 463
12 73 306
21 81 439
27 36 223
12 4 622
28 81 656
37 45 57
42 70 230
15 70 720
17 8 265
39 76 833
42 64 28
27 9 299
10 9 109
10 26 508
13 56 812
35 69 218
11 77 114
37 69 20
10 45 578
13 77 722
8 79 312
33 58 642
15 47 920
7 17 750
41 65 356
1 33 92
38 7 173
39 60 126
0 76 306
4 67 654
35 40 26
12 48 584
1 61 894
34 22 891
1 3 907
39 36 464
34 9 471
44 3 433
8 33 309
10 7 474
48 2 415
29 72 397
12 69 222
39 8 360
45 64 920
26 2 292
3 36 605
29 9 84
7 81 450
35 32 713
1 77 476
25 4 144
13 68 496
12 26 323
39 67 313
10 17 594
9 76 619
48 3 598
10 79 118
4 4 551
22 10 256
26 61 353
20 68 818
4 51 44
39 42 563
30 69 158
47 19 724
38 4 166
27 4 705
40 47 518
29 39 203
2 20 875
45 14 872
44 35 409
14 82 570
25 40 375
47 52 868
18 67 740
7 28 513
25 61 70
23 46 366
33 34 20
29 56 352
42 18 514
40 73 739
27 34 53
7 84 252
32 81 307
17 58 200
17 64 220
19 35 873
1 74 882
38 43 213
14 42 129
20 42 644
2 67 444
25 4 504
37 35 578
0 22 559
42 79 367
16 8 275
38 79 14
20 31 819
23 70 596
47 62 745
27 74 483
5 33 129
10 81 81
38 58 54
25 81 815
30 4 45
8 72 403
11 17 144
38 9 400
47 4 503
7 45 555
25 9 447
30 14 212
41 18 399
39 63 306
37 11 868
11 80 214
13 58 923
25 76 79
6 16 841
47 46 139
17 77 704
30 72 379
36 4 47
40 45 667
4 70 272
36 4 128
1 67 696
6 21 344
23 4 137
10 33 679
1 27 282
46 77 144
46 17 568
33 18 151
1 82 694
39 11 428
42 77 133
29 73 752
48 60 99
15 61 459
27 48 204
43 58 341
2 47 86
39 62 551
9 40 219
48 58 186
29 71 230
17 78 110
23 70 781
7 57 603
20 3 856
4 14 871
8 8 212
33 44 252
18 7 292
11 77 646
18 70 792
9 30 817
1 19 663
30 3 166
34 58 172
43 4 309
15 2 404
6 14 299
46 77 155
17 47 51
14 34 67
48 82 185
41 31 458
22 65 168
27 3 572
44 30 372
42 34 704
44 77 87
37 67 519
16 52 374
44 51 106
36 20 170
23 41 717
31 51 226
39 69 97
20 14 728
2 63 153
40 64 697
15 82 200
6 9 627
39 81 457
35 39 188
13 82 52
4 45 534
26 81 291
26 56 101
4 5 16
26 44 665
37 7 434
21 68 291
45 79 191
44 81 318
27 77 287
8 82 921
14 86 736
22 43 20
49 81 870
25 48 203
25 58 415
9 0 647
43 58 505
32 45 634
13 33 769
39 70 116
26 71 69
10 14 191
12 18 185
8 78 797
7 82 897
44 23 396
18 23 438
24 34 565
7 68 242
24 18 602
48 12 432
33 61 843
45 70 624
40 65 242
33 30 385
29 47 798
34 77 592
44 55 902
15 3 405
33 82 22
49 45 709
31 34 165
29 73 886
8 71 862
39 4 297
37 63 314
8 27 539
39 45 324
42 77 854
38 11 475
25 17 312
45 58 220
49 56 539
18 23 110
41 60 206
27 18 447
12 77 166
11 45 774
16 58 94
35 56 399
41 12 278
1 79 677
26 61 790
1 40 490
27 72 787
27 81 434
11 48 431
37 12 629
44 57 358
16 81 555
10 65 230
12 58 866
8 62 444
31 11 427
8 65 123
2 42 634
44 22 532
45 44 272
37 72 553
18 39 561
25 7 663
15 71 921
26 44 369
18 47 616
14 18 533
42 24 166
36 3 117
45 45 430
13 19 243
38 3 330
39 83 214
14 42 366
20 34 848
14 7 322
44 2 906
25 69 121
45 4 358
23 82 552
30 16 471
15 14 134
44 82 453
11 63 420
19 58 66
0 64 494
25 3 912
28 76 599
7 51 5
31 18 870
41 58 725
27 81 160
42 18 519
24 55 860
30 31 271
19 7 702
29 48 162
37 33 364
19 58 308
0 23 126
13 42 673
14 77 753
48 82 627
27 34 536
36 44 252
34 8 112
41 68 602
37 70 616
34 37 152
41 72 922
39 71 489
4 72 503
22 3 874
21 45 535
34 66 579
27 9 914
36 39 110
41 86 328
26 13 553
33 70 529
23 9 737
40 70 329
5 81 525
19 73 852
0 6 921
23 77 666
28 42 146
38 18 253
3 9 97
27 45 56
25 76 917
47 62 563
8 16 441
41 2 623
27 40 419
37 10 177
34 74 468
17 23 288
14 9 444
20 34 354
4 42 669
17 80 381
27 76 347
35 76 148
39 79 672
27 70 489
18 64 342
23 11 176
41 42 190
27 46 295
32 51 331
45 3 913
26 84 371
12 10 650
11 67 673
43 43 289
37 62 188
32 81 114
12 23 650
42 11 421
29 81 654
39 58 447
9 31 699
9 2 555
42 19 737
3 13 720
24 7 399
15 40 133
37 86 187
28 83 233
36 19 334
38 83 826
0 81 822
20 46 743
1 50 693
27 0 160
43 79 660
25 59 234
26 16 471
39 79 133
44 73 168
2 14 393
32 20 844
39 22 63
44 49 796
37 82 371
14 49 115
0 7 17
45 56 910
17 43 247
5 12 357
49 19 459
34 7 100
14 78 877
12 68 316
14 65 198
28 65 828
37 66 185
11 50 546
11 73 334
41 44 525
11 25 35
23 2 357
45 62 662
13 12 394
38 46 109
14 79 909
42 58 922
42 70 45
47 51 787
28 3 315
36 67 559
16 38 13
27 4 720
9 16 304
8 64 230
18 80 105
9 17 667
23 27 619
41 23 172
12 31 554
12 52 613
2 64 903
23 63 524
13 47 505
5 60 71
3 30 500
33 71 514
47 71 461
44 19 339
33 3 314
34 62 419
49 14 255
8 70 603
41 66 875
7 23 697
13 27 61
4 34 46
10 4 624
2 49 87
16 17 301
47 11 657
0 47 845
31 69 168
25 68 531
3 3 315
10 17 360
22 23 286
22 83 694
21 4 881
37 12 715
44 42 628
33 63 240
31 73 115
2 26 615
2 73 171
44 5 108
23 61 20
3 4 15
36 21 459
3 51 716
4 55 299
10 5 787
39 53 666
45 35 428
18 59 499
39 60 328
7 42 146
13 26 642
5 64 891
33 50 270
49 28 398
43 45 458
36 67 309
33 23 574
6 47 823
2 40 593
37 20 915
22 39 584
43 40 512
22 58 3
38 81 230
28 73 541
8 33 252
13 34 468
44 69 188
13 40 684
3 43 230
13 4 664
25 48 35
27 76 286
22 67 71
12 7 673
11 69 573
5 16 688
34 23 544
15 59 758
43 83 385
9 39 547
39 7 660
20 58 689
47 79 259
42 81 73
9 20 427
23 7 916
9 38 284
45 8 577
43 17 685
2 56 851
40 69 509
33 69 739
10 81 790
44 10 32
27 21 335
8 22 516
1 10 272
13 33 452
37 19 412
43 16 84
43 45 729
46 7 411
17 31 261
49 4 46
38 23 910
45 64 930
27 66 395
0 71 224
39 23 95
16 18 417
6 55 798
23 39 334
26 11 460
24 83 311
10 33 324
48 17 43
36 4 892
21 4 179
24 34 908
35 4 601
6 26 586
35 40 892
