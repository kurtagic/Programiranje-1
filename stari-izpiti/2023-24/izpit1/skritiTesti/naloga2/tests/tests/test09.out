[771, 0, 906, 913, 925, 875, 921, 916, 577, 914, 650, 868, 715, 720, 872, 0, 842, 825, 870, 929, 915, 459, 891, 910, 166, 35, 642, 619, 930, 0, 817, 879, 889, 769, 919, 873, 846, 152, 923, 584, 892, 877, 673, 746, 926, 926, 743, 920, 707, 796, 693, 787, 868, 666, 0, 902, 910, 603, 923, 758, 328, 894, 904, 524, 930, 828, 875, 740, 818, 846, 876, 921, 922, 907, 882, 0, 917, 910, 877, 909, 381, 897, 921, 826, 845, 0, 736]
